test/pytest/test_bridge.py

```diff
@@ -47,13 +47,13 @@ def send_open(self, payload, channel=None, **kwargs):
         self.send_json('', command='open', channel=channel, payload=payload, **kwargs)
         return channel
 
-    async def check_open(self, payload, channel=None, problem=None, **kwargs):
+    async def check_open(self, payload, channel=None, close_args: Optional[Dict[str, object]] = None, **kwargs):
```

Comment:
I feel like this approach makes it harder to use the function in the simple failure case while also not covering the case of checking for arguments in the `ready` message.  I checked, and not a lot of channel send arguments as part of ready, but at least dbus does.

Reply:
Reworked. Third time's the charm?